dvc/dagascii.py

```diff
@@ -45,7 +45,7 @@ def find_pager():
 
     logger.warning(
         "Unable to find `less` in the PATH. Check out "
-        "man.dvc.org/doc/command-reference/pipeline/show for more info."
```

Comment:
`man.dvc.org`  unrolls to `https://dvc.org/doc/command-reference/` so we would get 404 here